girder/models/item.py

```diff
@@ -438,6 +443,15 @@ def stream():
                 yield json.dumps(doc['meta'], default=str)
             yield (os.path.join(path, metadataFile), stream)
 
+    def _mimeFilter(self, file, mimeFilter):
+        """
+        Returns whether or not the given file should be passed through the given
+        MIME filter. If no MIME filter is specified, all files are allowed.
+        """
+        if not mimeFilter:
```

Comment:
This was by design, since it is assumed that nobody would request an empty set to download.